src/libsync/propagateremotemkdir.cpp

```diff
@@ -127,6 +126,10 @@ void PropagateRemoteMkdir::propfindResult(const QVariantMap &result)
     }
     if (result.contains("id")) {
         _item->_fileId = result["id"].toByteArray();
+        _item->_fileIdLocal = result["fileid"].toByteArray();
+        if (_item->_fileIdLocal.isEmpty()) {
+            _item->_fileIdLocal = _item->_fileId.left(8);
```

Comment:
I had it in a function, but then removed it when there was only a single use left. Then I later found this corner case (PropFind is only sent for servers <= 7.0.0). Yep, will put it into a function again.